.gitignore

```diff
@@ -4,7 +4,8 @@ node_modules
 ui/core/index.ts
 ui/*/*/index.html
 .dirstamp
-
+.vscode/launch.json
+.vscode/settings.json
```

Comment:
the settings json is actually our formatting setup so I dont think you want it in the gitignore because we want all our developers to have the same auto farm settings

Reply:
Also you should rever the deletion of the settings.json for sure

Reply:
Yeah its just that the settings.json doesnt have settings for auto formatting go, so it was messing with my vscode

Reply:
Fixed